character-controller.js

```diff
@@ -756,10 +757,10 @@ class StatePlayer extends PlayerBase {
     return this.getActionsState();
   }
   getActionsState() {

```

Comment:
the actions state happens inside the update loop. we actually want this to throw an error if it's being accessed before it's been initialized, otherwise we're hiding a deeper issue. so we want to make sure that we're just not calling getActionsState at all, because the update function has stopped, because the object that is updating has been destroyed